### ISSUE TITLE
Unable to produce Pdf with xelatex

### ISSUE THREAD

**Issue by rtrppl** [kawabata/ox-pandoc#66]
I'm unable to produce a Pdf with ox-pandoc, all other exports work as expected. When I use pandoc from the command line with the same arguments, i.e. ``pandoc -f org -t latex -o ~/Dropbox/org/2020_sose_PS_Außenpolitik.pdf --template=~/Dropbox/scripts/zh-notes.tex --pdf-engine=xelatex ~/Dropbox/org/2020_sose_PS_Außenpolitik.org``  it works perfectly fine, even with the template and xelatex (a combo I've been using for years). I have these two lines in my .emacs:

```
(require 'ox-pandoc)
(setq org-pandoc-options '((pdf-engine . "xelatex")(template . "~/Dropbox/scripts/zh-notes.tex")))
```
 

I'm new to emacs and org-mode, my apologies if I'm missing something obvious. Below is the messages buffer. I'm running Emacs 26.3, org-mode 9.3.6, pandoc 2.1.2, texlive 2020 on MacOS 10.14.6. [I have edited the directories for some privacy.]

```
Saving file ~/Dropbox/org/2020_sose_PS_Außenpolitik.tmpZYUhhi.org...
Wrote ~/Dropbox/org/2020_sose_PS_Außenpolitik.tmpZYUhhi.org
Running pandoc with args: (-f org -t latex -o ~/Dropbox/org/2020_sose_PS_Außenpolitik.pdf --template=~/Dropbox/scripts/zh-notes.tex --pdf-engine=xelatex ~/Dropbox/org/2020_sose_PS_Außenpolitik.tmpZYUhhi.org)
debug :: sentinel enter!
Error occured. 
ERROR: collate_byname<char>::collate_byname failed to construct for 

Sorry, but xelatex did not succeed.

You may want to visit the MiKTeX project page, if you need help.
libc++abi.dylib: terminating with uncaught exception of type MiKTeX::Core::MiKTeXException: MiKTeX encountered an internal error.
ERROR: collate_byname<char>::collate_byname failed to construct for 

Sorry, but xelatex did not succeed.

You may want to visit the MiKTeX project page, if you need help.
libc++abi.dylib: terminating with uncaught exception of type MiKTeX::Core::MiKTeXException: MiKTeX encountered an internal error.
ERROR: collate_byname<char>::collate_byname failed to construct for 

Sorry, but xelatex did not succeed.

You may want to visit the MiKTeX project page, if you need help.
libc++abi.dylib: terminating with uncaught exception of type MiKTeX::Core::MiKTeXException: MiKTeX encountered an internal error.
Error producing PDF.Saving file 
[...]org/2020_sose_PS_Außenpolitik.tmpZYUhhi.org...
Wrote [...]org/2020_sose_PS_Außenpolitik.tmpZYUhhi.org
Running pandoc with args: (-f org -t latex -o ~/Dropbox/org/2020_sose_PS_Außenpolitik.pdf --pdf-engine=xelatex ~/Dropbox/org/2020_sose_PS_Außenpolitik.tmpZYUhhi.org)
debug :: sentinel enter!
Error occured. 
ERROR: collate_byname<char>::collate_byname failed to construct for 

Sorry, but xelatex did not succeed.

You may want to visit the MiKTeX project page, if you need help.
libc++abi.dylib: terminating with uncaught exception of type MiKTeX::Core::MiKTeXException: MiKTeX encountered an internal error.
ERROR: collate_byname<char>::collate_byname failed to construct for 

Sorry, but xelatex did not succeed.

You may want to visit the MiKTeX project page, if you need help.
libc++abi.dylib: terminating with uncaught exception of type MiKTeX::Core::MiKTeXException: MiKTeX encountered an internal error.
ERROR: collate_byname<char>::collate_byname failed to construct for 

Sorry, but xelatex did not succeed.

You may want to visit the MiKTeX project page, if you need help.
libc++abi.dylib: terminating with uncaught exception of type MiKTeX::Core::MiKTeXException: MiKTeX encountered an internal error.
Error producing PDF.
```

**Comment by tarleb** [comment]
Could you check if the command *really* works if pasted into the terminal? The reason I'm asking is that pandoc does not expand the tilde `~` to `$HOME`. The expansion is done by the shell, but only if the tilde is at the beginning of the argument. `--template=~/foo.latex` and `--template ~/foo.latex` (note the missing `=` in the latter) usually mean very different things.

Does it help to use `expand-file-name`?

``` elisp
(setq org-pandoc-options
      `((pdf-engine . "xelatex")
        (template . ,(expand-file-name "~/Dropbox/scripts/zh-notes.tex"))))
```

**Comment by rtrppl** [comment]
Sorry for the confusion regarding the directories & tilde. I've manually replaced the correct folder name (which includes my real name) with the tilde. It was the exact same command working in the terminal but not working with emacs/ox-pandoc. I have, however, found a solution just a few minutes ago.

I thought that maybe my install of MiKTex was somehow broken and tried replacing it with the current one from https://miktex.org, which did not work (MikTex Console failed to load/install an update). After that I resolved to more drastic measures and replaced **Texlive** (80mb) with **MacTex** (4gb). Now everything works as it should (except for the bloat) and this issue can be closed. However, I'm still curious as to the reasons for all of this  - given that my setup was working flawlessly in the terminal.
